gui/src/app/SamplerOutputView/SamplerOutputView.tsx

```diff
@@ -108,7 +108,9 @@ const DrawsView: FunctionComponent<DrawsViewProps> = ({
   >(300);
   const draws2 = useMemo(() => {
```

Comment:
As long as you're changing things, might as well make this more semantic--`formattedDraws` or the like?